### PR TITLE
Add a new section to describe how to serve results and query data from RW

### PR DESCRIPTION
**Pull request by hengm3467**
Add a new section to talk about the different ways to retrieve data from RW.

### CODE REVIEW

File: serve/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+This section explains how to query and interact with data in RisingWave.
+
+RisingWave offers several methods for serving results, catering to various use cases, from ad-hoc analysis to application integration.
+
+## Query with `SELECT` statements
+
+Retrieve data directly from RisingWave using standard SQL SELECT queries against tables or materialized views. Use this method for ad-hoc analysis, exploring the latest results, and extracting specific data subsets.
+
+For syntax details, see [`SELECT`](/sql/commands/sql-select).
```
Comment (st1page) [inline serve/overview.mdx]: Can add link to the ad-hoc processing here https://github.com/risingwavelabs/risingwave-docs/blob/main/processing/overview.mdx#ad-hoc-on-read-vs-streaming-on-write

File: mint.json [hunk 2]
```diff
@@ -443,13 +445,22 @@
             "processing/time-travel-queries"
           ]
         },
+        {
+          "group": "Serve results",
```
Comment (st1page) [inline mint.json]: Not sure if we need to follow the other three items 👀 "Serve data"?

**Review by st1page — COMMENTED** [review]
LGTM!

File: serve/overview.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Connect to RisingWave with psql or any other PostgreSQL-compatible client to execute these queries. RisingWave is compatible with many data visualization tools. Here are a few that we have tested:
+
+- Beekeeper Studio
+- DBeaver
+- Grafana
+- Looker
+- Metabase
+- Superset
```
Comment (neverchanje) [inline serve/overview.mdx]:
```suggestion
Connect to RisingWave with psql or any other PostgreSQL-compatible client to execute these queries. RisingWave is compatible with [many data visualization tools](/integrations/visualization/overview). Here are a few that we have tested:

- [Beekeeper Studio](/integrations/visualization/beekeeper-studio)
- [DBeaver](/integrations/visualization/dbeaver)
- [Grafana](/integrations/visualization/grafana)
- [Looker](/integrations/visualization/looker)
- [Metabase](/integrations/visualization/metabase)
- [Superset](/integrations/visualization/superset)
```

**Review by neverchanje — APPROVED** [review]
Rest looks good to me.

File: mint.json (hunk 2 above)
Comment (hengm3467) [reply mint.json]: When we have both "Serve data" and "Deliver data", the distinction between these two are not alway clear. Maybe just "Query data"?

**Pull request merged**